db/migrate/24_localize_publish_column.rb

```diff
@@ -10,7 +10,7 @@ def up
 
     new_translations = Qbrick::Page.translated_columns_for(:published) - %w(published_en)
     Qbrick::Page.all.each do |page|
-      page.update Hash[new_translations.zip([page.published_en] * new_translations.count)]
+      page.update_columns Hash[new_translations.zip([page.published_en] * new_translations.count)]
```

Comment:
exactly, especially the new [`create_path`](https://github.com/screenconcept/qbrick/blob/master/app/models/qbrick/page.rb#L28) was the reason this migration broke on a legacy project because it was still called `url` in the db there. But we don't need the callbacks here at all (no need to generate the paths here again).